doc/guides/west/build-flash-debug.rst

```diff
@@ -19,6 +22,71 @@ build system commands with the same names directly delegate to West.
    .. contents::
       :local:
 
+.. _west-building:
+
+Building: ``west build``
+************************
+
+.. tip:: Run ``west build -h`` for additional help.
+
+The ``build`` command allows you to build any source tree from any directory
+in your file system, placing the build results in a folder of your choice.
+
+In its simplest form, the command can be run by navigating to the root folder
+(i.e. the folder containing a file:`CMakeLists.txt` file) of the Zephyr
+application of your choice and running::
+
+  west build -b <BOARD>
+
+Where ``<BOARD>`` is the name of the board you want to build for. This is
+exactly the same name you would supply to CMake if you were to invoke it with:
+``cmake -DBOARD=<BOARD>``.
+A build folder named :file:`./build` (default name) will be created and the
+build output will be placed in it.
+
+To specify the build directory, use ``--build-dir`` (or ``-d``)::
+
+  west build -b <BOARD> --build-dir path/to/build/directory
+
+Since the build directory defaults to :file:`./build`, if you do not specify
```

Comment:
Same here and elsewhere; I think the './' could be removed to make the output look a little cleaner